subiquity/ui/mount.py

```diff
@@ -2,6 +2,9 @@
 import os
 import re
 
+import gettext
+gettext.install('subiquity')
```

Comment:
does it make sense to do this in `subiquity/__init__.py`?

Reply:
I haven't tried running subiquity yet. And based on the docs ".install()" is new world order. It potentially makes sense to do in subiquitycore / first ever import, or possibly just at the top of the entry-point scripts. Also I'm not sure if we will need translations in the console-conf scripts, and if those need their own separate translation domain. I think next steps will be for me to get a top level script running with a translation. After that boot subiquity with a translation set in the environment. And then we can look at changing translations on the welcome screen.